spec/dummy/config/environments/test.rb

```diff
@@ -6,17 +8,18 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Turn false under Spring and add config.action_view.cache_template_loading = true.
   config.cache_classes = true
 
-  # Do not eager load code on boot. This avoids loading your whole application
-  # just for the purpose of running a single test. If you are using a tool that
-  # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  # Eager loading loads your whole application. When running a single test locally,
+  # this probably isn't necessary. It's a good idea to do in a continuous integration
+  # system, or in some way before deploying your code.
+  config.eager_load = ENV["CI"].present?
```

Comment:
This is the only one I'm not sure about, but I don't think it will hurt anything in reality.

Reply:
I agree.  I don't think it's a big deal.  It might lead to test contamination by loading things earlier than previously but who knows. It's a small dummy app.